webapp/app.py

```diff
@@ -170,6 +173,13 @@ def utility_processor():
 app.add_url_rule(
     "/security/<regex('(cve-|CVE-)\\d{4}-\\d{4,7}'):cve_id>", view_func=cve
 )
+app.add_url_rule("/security/cve", view_func=create_cve, methods=["POST"])
+app.add_url_rule("/security/cve", view_func=update_cve, methods=["PUT"])
```

Comment:
We agreed that we could support updating multiple CVEs at `/security/cve`, but what you seem to be offering here is to update a single CVE. This is not the right interface for that. If you want to offer updating a single CVE it should look like this:

``` python3
app.add_url_rule(
    "/security/<regex('(cve-|CVE-)\\d{4}-\\d{4,7}'):cve_id>",
    view_func=update_cve,
    methods=["PUT"]
)
```